### PR TITLE
The initial OBS import

### PR DESCRIPTION
**Pull request by lslezak**
- You can skip the first commit (the files in the `package/` subdir) - this is just a raw copy from https://build.opensuse.org/package/show/YaST:Head/patterns-yast
- This is mainly about integrating the package into our usual workflow (Travis and Jenkins)
- The pattern cleanup will follow later (there are still mentioned some already dropped packages)
- The Travis build fails for now as the global pattern change has not been accepted in Factory yet (there are some new RPM macros used)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "yast/rake"
+
+Yast::Tasks.configuration do |conf|
+  # lets ignore license check for now
+  conf.skip_license_check << /.*/
```
Comment (jreidinger) [inline Rakefile]: is it needed for this package?

Comment (lslezak) [reply Rakefile]: Um, I'm not sure actually, maybe not, I just want to be on the safe side. The plan is to [enable the license check](https://trello.com/c/YLWtoKFR/915-todo-run-rake-check-license-at-travis) for all Travis builds.

Comment (jreidinger) [reply Rakefile]: yeah, but this one actually skip license check for all files. which is probably what we do not want. So to try it, simply remove this line and run `rake check:license`

**Review by jreidinger — APPROVED** [review]
LGTM

**Pull request merged**